infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationMetadataManager.java

```diff
@@ -913,23 +889,6 @@ public void removeSession(TxnContext txn, LogReplicationSession session) {
         txn.delete(metadataTable, session);
     }
 
-    public void removeSession(LogReplicationSession session) {
```

Comment:
Why did we remove this?

Reply:
clarified offline that there is another API called `removeSession` that is used instead.